src/orion/client/__init__.py

```diff
@@ -187,6 +188,29 @@ def create_experiment(
     return ExperimentClient(experiment, producer, heartbeat)
 
 
+def get_experiment(name: str, version: int = None) -> ExperimentView:
```

Comment:
It should additionally support storage argument like in create_experiment. With `setup_storage` like [here](
https://github.com/Epistimio/orion/pull/439/files#diff-bf2b327fcde4615d08710f7e9c57188dR162).

Reply:
Good catch!